L3_S1.js

```diff
@@ -0,0 +1,90 @@
+// lecture3 - STEP1. 기본기능 구현
+
+class todoProgram {
+
+  // 변수 생성하기
+  constructor() {
+    this.task = [];
+    this.id = 1;
+  }
+
+  // 한줄띄어쓰는 함수
+  newline() {
+    console.log(``);
+  }
+
+  // 현재상태를 출력하는 함수
+  printNow() {
+    this.doneCount = 0;
+    this.doingCount = 0;
+    this.todoCount = 0;
+    for (let keys of this.task) {
```

Comment:
keys라고 하면 뭔지 읽기 어려운거 같네요.

이런식으로 바꿔보는 게 어때요? 

``
for (let taks of this.taskList) {
``

Reply:
```suggestion
    for (let task of this.taskList) {
```
이렇게 바꿔볼게요. taskList를 task로 잘못 쓰고 있었네요